incubator/monochart/values.yaml

```diff
@@ -0,0 +1,108 @@
+replicaCount: 1
+image:
+  repository: nginx
+  tag: 0.1.0
+  pullPolicy: IfNotPresent
+  pullSecrets:
+config:
+#   name: value
+env:
+  open:
```

Comment:
See above. I think we should have `configMap`, `secrets` and `inline` types

Reply:
@osterman good point, but I have some thoughts about this.
Configmap usually contains properties like `a.b.c: value` and this is not really an environment variable (it's looks like the set of config-files more)

Reply:
Yea, what about dropping the `env` top-level convention and instead doing:

```
configMaps:
  env:
    FOO: bar
   files:
     "/etc/blah.conf": "asd"
secrets:
 # same....
```

Reply:
yeap, this variant looks good

Reply:
I've pushed a PR to common chart with changes required to render env from configmaps.
But do we need dependency on common chart or monochart should have all required templates?